sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -359,6 +359,7 @@ class SparkSqlAstBuilder extends AstBuilder {
         ctx.identifier.getText.toLowerCase(Locale.ROOT) match {
           case "file" => AddFileCommand(maybePaths)
           case "jar" => AddJarCommand(maybePaths)
+          case "archive" => AddArchiveCommand(maybePaths)
```

Comment:
Can we update the docs above? -> ` Create a [[AddFileCommand]], [[AddJarCommand]], [[ListFilesCommand]] or  ... `